daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
```

Comment:
On Sat, Aug 22, 2015 at 11:45 AM, Madhu Venugopal <notifications@github.com>
wrote:

> In daemon/container_unix.go
> <https://github.com/docker/docker/pull/15747#discussion_r37696803>:
>
> > @@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
> >  		return err
> >  	}
> >
> > -	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
> > +	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
>
> @huikang <https://github.com/huikang> yes we can enhance libnetwork to
> satisfy such a need. But we have to make sure it fits the model properly so
> that the criu feature works seamlessly across all the plugins. The best way
> to to do is to bring in the criu related design into libnetwork core and
> have an API towards the driver that will support the use-case.
>
Thanks. I will think about this design of this API.


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/15747/files#r37696803>.
>